### DISCUSSION TITLE
How do I synchronously remove dependent modules and global entities for the current project

### DISCUSSION THREAD

**Discussion by tomoat** [pnpm/pnpm#4205] [Q&A] [answered]
When I use the `rm -rf node_modules ` command to delete the current project module, I cannot delete its corresponding hard link entity. Will this cause the global entity module to become bigger and bigger? How can you avoid this problem?

**Comment by tomoat** [comment]
I think this is very important, can you add it to the documentation or faQs?

**Comment by zkochan** [comment, marked as answer]
You can run [pnpm store prune](https://pnpm.io/cli/store#prune) to remove files that are not used from the global storage. 

But the global storage growth really slowly. I personally never run `pnpm store prune` and don't have issues with the size of the store.

**Reply by tomoat** [reply]
Thank you  @zkochan
